ci.sh

```diff
@@ -2,7 +2,18 @@
 
 set -ae
 
+if [[ ! -z "${TEAMCITY_VERSION}" ]]; then
+  echo "Running in TeamCity. Nope!"
+  exit 0
+fi
```

Comment:
Not needed, but is a good marker to see which PR you were inspired by 😄 .

Reply:
Haha, I'll remove it.